doc/api/child_process.markdown

```diff
@@ -758,13 +511,368 @@ throw.  The [`Error`][] object will contain the entire result from
   * `signal` {String} The signal used to kill the child process
   * `error` {Error} The error object if the child process failed or timed out
 
-`spawnSync` will not return until the child process has fully closed. When a
-timeout has been encountered and `killSignal` is sent, the method won't return
-until the process has completely exited. That is to say, if the process handles
-the `SIGTERM` signal and doesn't exit, your process will wait until the child
+The `child_process.spawnSync()` method is generally identical to
+`child_process.spawn()` with the exception that the function will not return
+until the child process has fully closed. When a timeout has been encountered
```

Comment:
Considered that but figured it would be better to keep things together a bit more. How about we look at this again once the larger set of changes have been landed?